modules/ppcp-settings/resources/js/Components/Screens/Onboarding/Steps/StepPaymentMethods.js

```diff
@@ -58,7 +61,13 @@ const StepPaymentMethods = () => {
 
 export default StepPaymentMethods;
 
-const PaymentStepTitle = () => {
+const PaymentStepTitle = ( ownBrandOnly ) => {
+	if ( ownBrandOnly.isBrandedOnly ) {
```

Comment:
Sorry, shouldn't this be just `ownBrandOnly` instead of `ownBrandOnly.isBrandedOnly` ?

Reply:
no, the object contains that isBrandedOnly that has the information I'm looking for.